<!DOCTYPE html>
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]--><!--[if gt IE 8]><!--><html lang="en" class="no-js"> <!--<![endif]-->
  <head>
    <meta charset="utf-8">
    <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" /><![endif]-->
    <title>Monther Abu-Remaileh's Profile | Stanford Profiles</title>
    <meta name="description" content="Monther Abu-Remaileh is part of Stanford Profiles, official site for faculty, postdocs, students and staff information (Expertise, Bio, Research, Publications, and more). The site facilitates research and collaboration in academic endeavors.">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
      
        <link rel="canonical" href="https://profiles.stanford.edu/monther-abu-remaileh;jsessionid=C0D8DFE61E2E8AE78408BBABD06C3B8E.cap-su-capappprd99">
      
    
    <link rel="stylesheet" href="/styles/main.css;jsessionid=C0D8DFE61E2E8AE78408BBABD06C3B8E.cap-su-capappprd99?r=9.11.0">
    <link rel="icon" href="/images/favicon.ico;jsessionid=C0D8DFE61E2E8AE78408BBABD06C3B8E.cap-su-capappprd99?r=9.11.0">
    <!--[if IE 7]><link rel="stylesheet" href="/styles/vendor/font-awesome/css/font-awesome-ie7.min.css;jsessionid=C0D8DFE61E2E8AE78408BBABD06C3B8E.cap-su-capappprd99?r=9.11.0"><![endif]-->
    <!-- Google Universal Analytics  (UA) -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    </script>
    <!-- End Google Universal Analytics  (UA) -->

    <!-- Google Analytics 4  (GA4) -->
    
      <script async src="https://www.googletagmanager.com/gtag/js?id=G-X6DK9R58FD"></script>
      <script>
        window.dataLayer = window.dataLayer || [];
        function gtag(){dataLayer.push(arguments);}
        gtag('js', new Date());
        gtag('config', 'G-X6DK9R58FD');
      </script>
    
    <!-- End Google Analytics 4  (GA4) -->
  </head>
  <body class="profile">
    <div class="wrapper  ">
      <header id="header">
        <div class="container navbar">
  
  <div class="column-logo brand ">
    <div class="logo">
      <div class="stanford-logo">
        <a href="http://www.stanford.edu/">
          <span>Stanford</span>
        </a>
      </div>
    </div>
    <div class="signature">
      <div class="site-name">
        <a href="/">
          <span class="site-name-1">Profiles</span>
        </a>
      </div>
    </div>
    
  </div>

  <div id="sidebarTrigger" class="pull-right">
    <a href="#" class="icon-background">
      <i class="icon-reorder"></i>
      <span class="sr-only">Menu</span>
    </a>
  </div>

  <nav class="navbar navbar-dropdown">
    <div class="search-header-region" id="searchHeaderContent"></div>

    <ul class="nav navbar-nav" aria-label="Primary Navigation">
      <li class="nav-item browse-menu has-submenu">
        <button class="nav-sub-menu-open">
          <i class="icon-list-ul"></i>
          <span>Browse</span>
          <i class="icon-caret-down"></i>
        </button>
        <div class="navbar navbar-accordion">
          <div class="browser" role="navigation">
  <ul class="nav navbar-nav unstyled">
    <li class="nav-item has-submenu dropdown">
      <button id="Schools" class="dropdown-toggle" data-toggle="dropdown">Schools</button>
      <ul class="dropdown-menu" role="menu" aria-labelledby="Schools">
          <li role="presentation"><a href="/browse/graduate-school-of-business" role="menuitem">Graduate School of Business</a></li>
          <li role="presentation"><a href="/browse/graduate-school-of-education" role="menuitem">Graduate School of Education</a></li>
          <li role="presentation"><a href="/browse/school-of-engineering" role="menuitem">School of Engineering</a></li>
          <li role="presentation"><a href="/browse/school-of-humanities-and-sciences" role="menuitem">School of Humanities and Sciences</a></li>
          <li role="presentation"><a href="/browse/school-of-medicine" role="menuitem">School of Medicine</a></li>
          <li role="presentation"><a href="/browse/stanford-doerr-school-of-sustainability" role="menuitem">Stanford Doerr School of Sustainability</a></li>
      </ul>
    </li>
    <li class="nav-item has-submenu dropdown">
      <button id="Institutes &amp; Research Centers" class="dropdown-toggle" data-toggle="dropdown">Institutes &amp; Research Centers</button>
      <ul class="dropdown-menu" role="menu" aria-labelledby="Institutes &amp; Research Centers">
          <li role="presentation"><a href="/browse/independent-labs-institutes-and-centers-dean-of-research/bio-x" role="menuitem">Bio-X</a></li>
          <li role="presentation"><a href="/browse/independent-labs-institutes-and-centers-dean-of-research/freeman-spogli-institute-for-international-studies" role="menuitem">Freeman Spogli Institute for International Studies</a></li>
          <li role="presentation"><a href="/browse/school-of-engineering/programs-centers-and-institutes/institute-for-computational-and-mathematical-engineering-icme" role="menuitem">Institute for Computational and Mathematical Engineering (ICME)</a></li>
          <li role="presentation"><a href="/browse/independent-labs-institutes-and-centers-dean-of-research/institute-for-human-centered-artificial-intelligence-hai" role="menuitem">Institute for Human-Centered Artificial Intelligence (HAI)</a></li>
          <li role="presentation"><a href="/browse/school-of-medicine/institute-for-stem-cell-biology-and-regenerative-medicine" role="menuitem">Institute for Stem Cell Biology and Regenerative Medicine</a></li>
          <li role="presentation"><a href="/browse/stanford-doerr-school-of-sustainability/precourt-institute-for-energy" role="menuitem">Precourt Institute for Energy</a></li>
          <li role="presentation"><a href="/browse/slac-national-accelerator-laboratory" role="menuitem">SLAC National Accelerator Laboratory</a></li>
          <li role="presentation"><a href="/browse/independent-labs-institutes-and-centers-dean-of-research/sarafan-chem-h" role="menuitem">Sarafan ChEM-H</a></li>
          <li role="presentation"><a href="/browse/independent-labs-institutes-and-centers-dean-of-research/stanford-institute-for-economic-policy-research-siepr" role="menuitem">Stanford Institute for Economic Policy Research (SIEPR)</a></li>
          <li role="presentation"><a href="/browse/independent-labs-institutes-and-centers-dean-of-research/stanford-pulse-institute" role="menuitem">Stanford PULSE Institute</a></li>
          <li role="presentation"><a href="/browse/stanford-doerr-school-of-sustainability/stanford-woods-institute-for-the-environment" role="menuitem">Stanford Woods Institute for the Environment</a></li>
          <li role="presentation"><a href="/browse/independent-labs-institutes-and-centers-dean-of-research/life-science-institutes-independent-labs-institutes-centers-dor/wu-tsai-human-performance-alliance" role="menuitem">Wu Tsai Human Performance Alliance</a></li>
          <li role="presentation"><a href="/browse/independent-labs-institutes-and-centers-dean-of-research/wu-tsai-neurosciences-institute" role="menuitem">Wu Tsai Neurosciences Institute</a></li>
      </ul>
    </li>
    <li class="nav-item has-submenu dropdown">
      <button id="Administration" class="dropdown-toggle" data-toggle="dropdown">Administration</button>
      <ul class="dropdown-menu" role="menu" aria-labelledby="Administration">
          <li role="presentation"><a href="/browse/office-of-external-relations" role="menuitem">Office of External Relations</a></li>
          <li role="presentation"><a href="/browse/office-of-vp-for-university-human-resources" role="menuitem">Office of VP for University Human Resources</a></li>
          <li role="presentation"><a href="/browse/office-of-vice-president-for-business-affairs-and-chief-financial-officer" role="menuitem">Office of Vice President for Business Affairs and Chief Financial Officer</a></li>
          <li role="presentation"><a href="/browse/stanford-university-libraries" role="menuitem">Stanford University Libraries</a></li>
          <li role="presentation"><a href="/browse/vice-provost-and-dean-of-research" role="menuitem">Vice Provost and Dean of Research</a></li>
          <li role="presentation"><a href="/browse/vice-provost-for-student-affairs" role="menuitem">Vice Provost for Student Affairs</a></li>
          <li role="presentation"><a href="/browse/vice-provost-for-undergraduate-education" role="menuitem">Vice Provost for Undergraduate Education</a></li>
      </ul>
    </li>


  </ul>
</div>

        </div>
      </li>

      
        <li class="nav-item signin-link">
          <a data-toggle="tooltip" href="/intranet/monther-abu-remaileh" class="sign-in-link" data-bypass>
            <i class="icon-signin visible-phone"></i>
            <span>Sign In</span>
          </a>
        </li>
        <li class="nav-item edit-profile-link">
          <a id="editProfile" href="https://cap.stanford.edu/profiles/auth/edit">
            <i class="icon-pencil visible-phone"></i>
            <span>Edit My Profile</span>
          </a>
        </li>

      

    </ul>
  </nav>

</div>

  
  
  
  
  
  
  
  
  
  
  
  
  
  

      </header>
      <main id="bodyContent">
        <div id="mainContent">
          <section id="mastheadContainer" class="banner profile">
  <div class="container">
  <div class="row-fluid">
    <div class="span12">
      <div class="image-holder">
        <img src="https://profiles.stanford.edu/proxy/api/cap/profiles/220380/resources/profilephoto/350x350.1564102873757.jpg" alt="Monther Abu-Remaileh" data-empty-src="/images/placeholder-350x350.png">
      </div>
      <div class="nameAndTitle">
        <h1>Monther Abu-Remaileh</h1>
        <h2>Assistant Professor of Chemical Engineering and of Genetics<br></h2>
        <div class="leave">
          <span class="leave-info">  </span>
          <span class="leave-note">  </span>
        </div>

      </div>
      <ul class="unstyled">
          <li>
            <a href="https://cap.stanford.edu/profiles/frdActionServlet?choiceId=printerprofile&amp;profileversion=full&amp;profileId=220380" target="_blank">
              <div class="icon-background"><i class="icon-print"></i></div>
              Print Profile
            </a>
          </li>
        <li>
          <a href="mailto:?subject=Monther%20Abu-Remaileh's%20Profile&amp;body=I%20would%20like%20to%20share%20the%20profile%20of%20Monther%20Abu-Remaileh%20with%20you.%20https://profiles.stanford.edu/monther-abu-remaileh" data-bypass>
            <div class="icon-background"><i class="icon-envelope-alt"></i></div>
            Email Profile
          </a>
        </li>
          <li>
            <a data-toggle="tooltip" href="/intranet/monther-abu-remaileh" data-bypass>
                <div class="icon-background"><i class="icon icon-lock"></i></div>
                View Stanford-only Profile
            </a>
          </li>
      </ul>
    </div>
  </div>
</div>

</section>
<section id="tabbedLayoutContainer">
  <div class="container breakout-phone">
  <div class="tab-dropdown">
    <a class="collapsed" data-toggle="collapse" data-target=".nav-collapse">Tab Menu <span class="selected-tab"></span><i class="icon-caret-down"></i></a>
  </div>
  <div class="nav-collapse collapse">
    <ul class="nav nav-tabs tabs4">
        <li class="active">
          <a id="bioTabLink" href="/monther-abu-remaileh?tab=bio" class="tab-link no-trigger">Bio</a>
        </li>
        <li class="">
          <a id="researchAndScholarshipTabLink" href="/monther-abu-remaileh?tab=research-and-scholarship" class="tab-link no-trigger">Research &amp; Scholarship</a>
        </li>
        <li class="">
          <a id="teachingTabLink" href="/monther-abu-remaileh?tab=teaching" class="tab-link no-trigger">Teaching</a>
        </li>
        <li class="">
          <a id="publicationsTabLink" href="/monther-abu-remaileh?tab=publications" class="tab-link no-trigger">Publications</a>
        </li>
    </ul>
  </div>
</div>
<div class="container breakout-phone">
  <div class="tab-content">
      <div id="bio" class="card tab-pane active">
        <div class="row-fluid">
  <div class="span8">
      <div id="bioContent" class="content-section">
        <h3>Bio</h3>
<hr>
<p>The Abu-Remaileh Lab is interested in identifying novel pathways that enable cellular and organismal adaptation to metabolic stress and changes in environmental conditions. We also study how these pathways go awry in human diseases such as cancer, neurodegeneration and metabolic syndrome, in order to engineer new therapeutic modalities.<br><br>To address these questions, our lab uses a multidisciplinary approach to study the biochemical functions of the lysosome in vitro and in vivo. Lysosomes are membrane-bound compartments that degrade macromolecules and clear damaged organelles to enable cellular adaptation to various metabolic states. Lysosomal function is critical for organismal homeostasis&#8212;mutations in genes encoding lysosomal proteins cause severe human disorders known as lysosomal storage diseases, and lysosome dysfunction is implicated in age-associated diseases including cancer, neurodegeneration and metabolic syndrome.<br><br>By developing novel tools and harnessing the power of metabolomics, proteomics and functional genomics, our lab will define 1) how the lysosome communicates with other cellular compartments to fulfill the metabolic demands of the cell under various metabolic states, 2) and how its dysfunction leads to rare and common human diseases. Using insights from our research, we will engineer novel therapies to modulate the pathways that govern human disease.</p>
      </div>




      <div id="academicAppointmentsContent" class="content-section">
        <h3>Academic Appointments</h3>
<hr>
<ul class="section-listing">
    <li class="section-list-item">
  <div class="description bulleted">Assistant Professor, <a href="http://cheme.stanford.edu/"> Chemical Engineering</a></div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Assistant Professor, Genetics</div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Member, <a href="http://biox.stanford.edu/"> Bio-X</a></div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Member, <a href="http://chri.stanford.edu/"> Maternal &amp; Child Health Research Institute (MCHRI)</a></div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Institute Scholar, <a href="https://chemh.stanford.edu/"> Sarafan ChEM-H</a></div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Member, <a href="http://cancer.stanford.edu"> Stanford Cancer Institute</a></div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Member, <a href="https://neuroinstitute.stanford.edu"> Wu Tsai Neurosciences Institute</a></div>
</li>
</ul>
      </div>



      <div id="honorsAndAwardsContent" class="content-section">
        <h3>Honors &amp; Awards</h3>
<hr>
<ul class="section-listing">
    <li class="section-list-item">
  <div class="description bulleted">The NIH Director's New Innovator Award, NIH (2021)</div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Cancer Innovation Award, Stanford Cancer Institute (2020)</div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Terman Faculty Fellow, Stanford University (2019)</div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Innovators Under 35 MENA, MIT Technology Review (2018)</div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">NCL-Stiftung Foundation Research Award, NCL-Stiftung (2018)</div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">The Charles A. King Trust Award, The Medical Foundation (2018)</div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">The EMBO Fellowship, EMBO (2014- 2016)</div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Adams Fellowship, Israel Academy of Sciences and Humanities (2009-2013)</div>
</li>
</ul>
      </div>




      <div id="professionalEducationContent" class="content-section">
        <h3>Professional Education</h3>
<hr>
<ul class="section-listing">
    <li class="section-list-item">
  <div class="description bulleted">Molecular Genetics, The Hebrew University of Jerusalem, Gene Regulation in Development and Cancer  (2014)</div>
</li>
    <li class="section-list-item">
  <div class="description bulleted">Postdoctoral training, Whitehead Institute/ MIT, Subcellular Metabolism  (2019)</div>
</li>
</ul>
      </div>








  </div>

  <div class="span4 side-bar">
      <div id="contactInfoContent" class="content-section contact-info">
        <h3>Contact</h3>
<hr>
<span style="display:none;" data-id="view__tabs__bio__concact_info_section"></span>
    <ul class="unstyled section-listing use-link-icons">
      <li class="primary">
            <div class="contact-info ">
                  <span><strong>Academic</strong></span>
                      <div class="extra-bottom-padding"><a data-bypass href="mailto:monther@stanford.edu" target="_top">monther@stanford.edu</a></div>
                  <div class="extra-bottom-padding">
                    <span class="affiliation">University - Faculty</span>
                      <span class="department"> Department:&amp;nbspChemical Engineering</span>
                      <span class="position extra-bottom-padding"> Position:&#160;Asst Professor</span>
                  </div>
            </div>
        </li>
    </ul>

<ul class="unstyled section-listing use-link-icons">

</ul>




      </div>

        <div id="additionalInfoContent" class="content-section additional-info">
              <h3>Additional Info</h3>
    <hr>
    <span style="display:none;" data-id="view__tabs__bio__additional_info_section"></span>
    <ul class="unstyled section-listing use-link-icons contact-info">
        <li>
            <div class="row no-margin">
                <span class="section23"> Mail Code:</span>
                <span class="section75 "> 4300</span>
            </div>
        </li>
    </ul>

        </div>

      <div id="linksContent" class="content-section links">
        <h3>Links</h3>
<hr>
<ul class="unstyled section-listing use-link-icons">
    <li class="external-link"><a href="http://www.abu-remaileh.com" class="Lab Site" target="_blank" data-bypass>Lab website</a></li>
</ul>

      </div>
  </div>

</div>

      </div>
      <div id="research-and-scholarship" class="card tab-pane ">
        <div class="row-fluid">
  <div class="span8">

      <div id="currentResearchAndScholarlyInterestsContent" class="content-section">
        <h3>Current Research and Scholarly Interests</h3>
<hr>
<p>We study the role of the lysosome in metabolic adaptation using subcellular omics approaches, functional genomics and innovative biochemical tools. We apply this knowledge to understand how lysosomal dysfunction leads to human diseases including neurodegeneration, cancer and metabolic syndrome.</p>
      </div>







  </div>
</div>
      </div>
      <div id="teaching" class="card tab-pane ">
        <div class="row-fluid">
  <div class="span8">
      <div id="coursesContent" class="content-section">
        <h3>2022-23 Courses</h3>
<hr>
<ul class="section-listing courses">
    <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20185B:%20Chemical%20Engineering%20Laboratory%20B&amp;academicYear=20222023">Chemical Engineering Laboratory B</a><br><span>CHEMENG 185B (Spr)</span></li>
    <li class="unstyled">
      <a class="independent-studies-label">Independent Studies (9)</a>
      <ul class="section-listing independent-studies">
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=BIOE%20191:%20Bioengineering%20Problems%20and%20Experimental%20Investigation&amp;academicYear=20222023">Bioengineering Problems and Experimental Investigation</a><br><span>BIOE 191 (Aut, Spr)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=BIOPHYS%20399:%20Directed%20Reading%20in%20Biophysics&amp;academicYear=20222023">Directed Reading in Biophysics</a><br><span>BIOPHYS 399 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=NEPR%20299:%20Directed%20Reading%20in%20Neurosciences&amp;academicYear=20222023">Directed Reading in Neurosciences</a><br><span>NEPR 299 (Aut)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=BIOPHYS%20300:%20Graduate%20Research&amp;academicYear=20222023">Graduate Research</a><br><span>BIOPHYS 300 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CBIO%20399:%20Graduate%20Research&amp;academicYear=20222023">Graduate Research</a><br><span>CBIO 399 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20600:%20Graduate%20Research%20in%20Chemical%20Engineering&amp;academicYear=20222023">Graduate Research in Chemical Engineering</a><br><span>CHEMENG 600 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=BIO%20300X:%20Out-of-Department%20Graduate%20Research&amp;academicYear=20222023">Out-of-Department Graduate Research</a><br><span>BIO 300X (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20190H:%20Undergraduate%20Honors%20Research%20in%20Chemical%20Engineering&amp;academicYear=20222023">Undergraduate Honors Research in Chemical Engineering</a><br><span>CHEMENG 190H (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20190:%20Undergraduate%20Research%20in%20Chemical%20Engineering&amp;academicYear=20222023">Undergraduate Research in Chemical Engineering</a><br><span>CHEMENG 190 (Aut, Win, Spr, Sum)</span></li>
      </ul>
    </li>
    <li class="unstyled">
      <a class="prior-year-courses-label">Prior Year Courses</a>
      <div class="prior-year-courses">
          <h4>2021-22 Courses</h4>
          <ul class="section-listing courses">
              <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20185A:%20Chemical%20Engineering%20Laboratory%20A&amp;academicYear=20212022">Chemical Engineering Laboratory A</a><br><span>CHEMENG 185A (Win)</span></li>
              <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20185B:%20Chemical%20Engineering%20Laboratory%20B&amp;academicYear=20212022">Chemical Engineering Laboratory B</a><br><span>CHEMENG 185B (Spr)</span></li>
          </ul>
          <h4>2020-21 Courses</h4>
          <ul class="section-listing courses">
              <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20185A:%20Chemical%20Engineering%20Laboratory%20A&amp;academicYear=20202021">Chemical Engineering Laboratory A</a><br><span>CHEMENG 185A (Win)</span></li>
              <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20185B:%20Chemical%20Engineering%20Laboratory%20B&amp;academicYear=20202021">Chemical Engineering Laboratory B</a><br><span>CHEMENG 185B (Spr)</span></li>
              <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20299:%20Graduate%20Practical%20Training&amp;academicYear=20202021">Graduate Practical Training</a><br><span>CHEMENG 299 (Sum)</span></li>
          </ul>
          <h4>2019-20 Courses</h4>
          <ul class="section-listing courses">
              <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CHEMENG%20299:%20Graduate%20Practical%20Training&amp;academicYear=20192020">Graduate Practical Training</a><br><span>CHEMENG 299 (Sum)</span></li>
          </ul>
      </div>
    </li>
  </ul>
      </div>
    
      <div id="stanfordAdviseesContent" class="content-section">
        <h3>Stanford Advisees</h3>
<hr>
<ul class="section-listing">
    <li class="advisees-list-item">
	  <span class="advisees-sub-heading">Med Scholar Project Advisor</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/aswini-krishnan">Aswini Krishnan</a></span>
</li>

    <li class="advisees-list-item">
	  <span class="advisees-sub-heading">Doctoral Dissertation Reader (AC)</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/jake-hsu">Jake Hsu</a>, <a href="https://profiles.stanford.edu/elise-loppinet">Elise Loppinet</a>, <a href="https://profiles.stanford.edu/vincent-masto">Vincent Masto</a>, <a href="https://profiles.stanford.edu/magdalena-murray">Magdalena Murray</a></span>
</li>

    <li class="advisees-list-item">
	  <span class="advisees-sub-heading">Postdoctoral Faculty Sponsor</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/wentao-dong">Wentao Dong</a>, <a href="https://profiles.stanford.edu/ali-ghoochani">Ali Ghoochani</a>, <a href="https://profiles.stanford.edu/jian-xiong">Jian Xiong</a></span>
</li>

    <li class="advisees-list-item">
	  <span class="advisees-sub-heading">Doctoral Dissertation Advisor (AC)</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/aswini-krishnan">Aswini Krishnan</a>, <a href="https://profiles.stanford.edu/226746">Cindy Lin</a>, <a href="https://profiles.stanford.edu/uche-medoh">Uche Medoh</a>, <a href="https://profiles.stanford.edu/austin-murchison">Austin Murchison</a>, Kwamina Nyame, <a href="https://profiles.stanford.edu/samantha-scharenberg">Sam Scharenberg</a>, George Walters-Marrah</span>
</li>

    <li class="advisees-list-item">
	  <span class="advisees-sub-heading">Doctoral Dissertation Co-Advisor (AC)</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/sai-gourisankar">Sai Gourisankar</a></span>
</li>

    <li class="advisees-list-item">
	  <span class="advisees-sub-heading">Postdoctoral Research Mentor</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/wentao-dong">Wentao Dong</a>, <a href="https://profiles.stanford.edu/ali-ghoochani">Ali Ghoochani</a>, <a href="https://profiles.stanford.edu/jian-xiong">Jian Xiong</a></span>
</li>

    <li class="advisees-list-item">
	  <span class="advisees-sub-heading">Doctoral (Program)</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/austin-murchison">Austin Murchison</a></span>
</li>

</ul>
      </div>
    
      <div id="graduateAndFellowshipProgramsContent" class="content-section">
        <h3>Graduate and Fellowship Programs</h3>
<hr>
<ul class="section-listing">
    <li class="section-list-item">
  <div class="description bulleted"><a href="http://medcatalog.stanford.edu/php/dept.php?DNo=625">Biophysics (Phd Program)</a></div>
</li>
    <li class="section-list-item">
  <div class="description bulleted"><a href="http://medcatalog.stanford.edu/php/dept.php?DNo=730">Cancer Biology (Phd Program)</a></div>
</li>
    <li class="section-list-item">
  <div class="description bulleted"><a href="http://medcatalog.stanford.edu/php/dept.php?DNo=770">Genetics (Phd Program)</a></div>
</li>
    <li class="section-list-item">
  <div class="description bulleted"><a href="http://medcatalog.stanford.edu/php/dept.php?DNo=890">Neurosciences (Phd Program)</a></div>
</li>
</ul>
      </div>
    
  </div>
</div>
      </div>
      <div id="publications" class="card tab-pane ">
        
<div class="row-fluid">
  <div class="span8">
    <div id="featuredPublicationsContent"></div>
    <div id="allPublicationsContent">
        <h3>All Publications</h3>
<hr>
<ul class="section-listing articles" data-count="30">
    <li class="publication featured article">
  <cite><span class="title">
  <span>Lysosomal cathepsin D mediates endogenous mucin glycodomain catabolism in mammals.</span>
  <i>Proceedings of the National Academy of Sciences of the United States of America</i>
</span>
<span class="authors">Pedram, K., Laqtom, N. N., Shon, D. J., Di Spiezio, A., Riley, N. M., Saftig, P., Abu-Remaileh, M., Bertozzi, C. R.</span>

<span class="details">
   <span class="year">2022</span>; <span class="volume">119 (39)</span><span class="pages">: e2117105119</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Mucins are functionally implicated in a range of human pathologies, including cystic fibrosis, influenza, bacterial endocarditis, gut dysbiosis, and cancer. These observations have motivated the study of mucin biosynthesis as well as the development of strategies for inhibition of mucin glycosylation. Mammalian pathways for mucin catabolism, however, have remained underexplored. The canonical view, derived from analysis of N-glycoproteins in human lysosomal storage disorders, is that glycan degradation and proteolysis occur sequentially. Here, we challenge this view by providing genetic and biochemical evidence supporting mammalian proteolysis of heavily O-glycosylated mucin domains without prior deglycosylation. Using activity screening coupled with mass spectrometry, we ascribed mucin-degrading activity in murine liver to the lysosomal protease cathepsin D. Glycoproteomics of substrates digested with purified human liver lysosomal cathepsin D provided direct evidence for proteolysis within densely O-glycosylated domains. Finally, knockout of cathepsin D in a murine model of the human lysosomal storage disorder neuronal ceroid lipofuscinosis 10 resulted in accumulation of mucins in liver-resident macrophages. Our findings imply that mucin-degrading activity is a component of endogenous pathways for glycoprotein catabolism in mammalian tissues.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1073/pnas.2117105119">DOI 10.1073/pnas.2117105119</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/36122205">PubMedID 36122205</a></span>
  </p>
      
    </div>
</li>
    <li class="publication featured article">
  <cite><span class="title">
  <span>CLN3 is required for the clearance of glycerophosphodiesters from lysosomes.</span>
  <i>Nature</i>
</span>
<span class="authors">Laqtom, N. N., Dong, W., Medoh, U. N., Cangelosi, A. L., Dharamdasani, V., Chan, S. H., Kunchok, T., Lewis, C. A., Heinze, I., Tang, R., Grimm, C., Dang Do, A. N., Porter, F. D., Ori, A., Sabatini, D. M., Abu-Remaileh, M.</span>

<span class="details">
   <span class="year">2022</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Lysosomes have many roles, including degrading macromolecules and signalling to the nucleus1. Lysosomal dysfunction occurs in various human conditions, such as common neurodegenerative diseases and monogenic lysosomal storage disorders (LSDs)2-4. For most LSDs, the causal genes have been identified but, in some, the function of the implicated gene is unknown, in part because lysosomes occupy a small fraction of the cellular volume so that changes in lysosomal contents are difficult to detect. Here we develop the LysoTag mouse for the tissue-specific isolation of intact lysosomes that are compatible with the multimodal profiling of their contents. We used the LysoTag mouse to study CLN3, a lysosomal transmembrane protein with an unknown function. In children, the loss of CLN3 causes juvenile neuronal ceroid lipofuscinosis (Batten disease), a lethal neurodegenerative LSD. Untargeted metabolite profiling of lysosomes from the brains of mice lacking CLN3 revealed a massive accumulation of glycerophosphodiesters (GPDs)-the end products of glycerophospholipid catabolism. GPDs also accumulate in the lysosomes of CLN3-deficient cultured cells and we show that CLN3 is required for their lysosomal egress. Loss of CLN3 also disrupts glycerophospholipid catabolism in the lysosome. Finally, we found elevated levels of glycerophosphoinositol in the cerebrospinal fluid of patients with Batten disease, suggesting the potential use of glycerophosphoinositol as a disease biomarker. Our results show that CLN3 is required for the lysosomal clearance of GPDs and reveal Batten disease as a neurodegenerative LSD with a defect in glycerophospholipid metabolism.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1038/s41586-022-05221-y">DOI 10.1038/s41586-022-05221-y</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/36131016">PubMedID 36131016</a></span>
  </p>
      
    </div>
</li>
    <li class="publication featured article">
  <cite><span class="title">
  <span>Isotope tracing in health and disease.</span>
  <i>Current opinion in biotechnology</i>
</span>
<span class="authors">Dong, W., Rawat, E. S., Stephanopoulos, G., Abu-Remaileh, M.</span>

<span class="details">
   <span class="year">2022</span>; <span class="volume">76</span><span class="pages">: 102739</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Biochemical characterization of metabolism provides molecular insights for understanding biology in health and disease. Over the past decades, metabolic perturbations have been implicated in cancer, neurodegeneration, and diabetes, among others. Isotope tracing is a technique that allows tracking of labeled atoms within metabolites through biochemical reactions. This technique has become an integral component of the contemporary metabolic research. Isotope tracing measures substrate contribution to downstream metabolites and indicates its utilization in cellular metabolic networks. In addition, isotopic labeling data are necessary for quantitative metabolicflux analysis. Here, we review recent work utilizing metabolic tracing to study health and disease, and highlight its application to interrogate subcellular, intercellular, and in vivo metabolism. We further discuss the current challenges and opportunities to expand the utility of isotope tracing to new research areas.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1016/j.copbio.2022.102739">DOI 10.1016/j.copbio.2022.102739</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/35738210">PubMedID 35738210</a></span>
  </p>
      
    </div>
</li>
    <li class="publication featured article">
  <cite><span class="title">
  <span>Lessons from metabolic perturbations in lysosomal storage disorders for neurodegeneration</span>
  <i>CURRENT OPINION IN SYSTEMS BIOLOGY</i>
</span>
<span class="authors">Medoh, U. N., Chen, J. Y., Abu-Remaileh, M.</span>

<span class="details">
   <span class="year">2022</span>; <span class="volume">29</span>
</span></cite>
    <div class="detail">
      <p class="doi">
    <span>View details for <a href="https://doi.org/10.1016/j.coisb.2021.100408">DOI 10.1016/j.coisb.2021.100408</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000853279900003">Web of Science ID 000853279900003</a></span>
  </p>
      
    </div>
</li>
    <li class="publication featured article">
  <cite><span class="title">
  <span>Lysosomal metabolomics reveals V-ATPase- and mTOR-dependent regulation of amino acid efflux from lysosomes.</span>
  <i>Science (New York, N.Y.)</i>
</span>
<span class="authors">Abu-Remaileh, M. n., Wyant, G. A., Kim, C. n., Laqtom, N. N., Abbasi, M. n., Chan, S. H., Freinkman, E. n., Sabatini, D. M.</span>

<span class="details">
   <span class="year">2017</span>; <span class="volume">358 (6364)</span><span class="pages">: 807&#8211;13</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">The lysosome degrades and recycles macromolecules, signals to the cytosol and nucleus, and is implicated in many diseases. Here, we describe a method for the rapid isolation of mammalian lysosomes and use it to quantitatively profile lysosomal metabolites under various cell states. Under nutrient-replete conditions, many lysosomal amino acids are in rapid exchange with those in the cytosol. Loss of lysosomal acidification through inhibition of the vacuolar H+-adenosine triphosphatase (V-ATPase) increased the luminal concentrations of most metabolites but had no effect on those of the majority of essential amino acids. Instead, nutrient starvation regulates the lysosomal concentrations of these amino acids, an effect we traced to regulation of the mechanistic target of rapamycin (mTOR) pathway. Inhibition of mTOR strongly reduced the lysosomal efflux of most essential amino acids, converting the lysosome into a cellular depot for them. These results reveal the dynamic nature of lysosomal metabolites and that V-ATPase- and mTOR-dependent mechanisms exist for controlling lysosomal amino acid efflux.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1126/science.aan6298">DOI 10.1126/science.aan6298</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/29074583">PubMedID 29074583</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5704967">PubMedCentralID PMC5704967</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Small molecule C381 targets the lysosome to reduce inflammation and ameliorate disease in models of neurodegeneration.</span>
  <i>Proceedings of the National Academy of Sciences of the United States of America</i>
</span>
<span class="authors">Vest, R. T., Chou, C. C., Zhang, H., Haney, M. S., Li, L., Laqtom, N. N., Chang, B., Shuken, S., Nguyen, A., Yerra, L., Yang, A. C., Green, C., Tanga, M., Abu-Remaileh, M., Bassik, M. C., Frydman, J., Luo, J., Wyss-Coray, T.</span>

<span class="details">
   <span class="year">2022</span>; <span class="volume">119 (11)</span><span class="pages">: e2121609119</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">SignificanceNeurodegenerative diseases are poorly understood and difficult to treat. One common hallmark is lysosomal dysfunction leading to the accumulation of aggregates and other undegradable materials, which cause damage to brain resident cells. Lysosomes are acidic organelles responsible for breaking down biomolecules and recycling their constitutive parts. In this work, we find that the antiinflammatory and neuroprotective compound, discovered via a phenotypic screen, imparts its beneficial effects by targeting the lysosome and restoring its function. This is established using a genome-wide CRISPRi target identification screen and then confirmed using a variety of lysosome-targeted studies. The resulting small molecule from this study represents a potential treatment for neurodegenerative diseases as well as a research tool for the study of lysosomes in disease.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1073/pnas.2121609119">DOI 10.1073/pnas.2121609119</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/35259016">PubMedID 35259016</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Ferroptosis regulation by the NGLY1/NFE2L1 pathway.</span>
  <i>Proceedings of the National Academy of Sciences of the United States of America</i>
</span>
<span class="authors">Forcina, G. C., Pope, L., Murray, M., Dong, W., Abu-Remaileh, M., Bertozzi, C. R., Dixon, S. J.</span>

<span class="details">
   <span class="year">2022</span>; <span class="volume">119 (11)</span><span class="pages">: e2118646119</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">SignificanceFerroptosis is an oxidative form of cell death whose biochemical regulation remains incompletely understood. Cap'n'collar (CNC) transcription factors including nuclear factor erythroid-2-related factor 1 (NFE2L1/NRF1) and NFE2L2/NRF2 can both regulate oxidative stress pathways but are each regulated in a distinct manner, and whether these two transcription factors can regulate ferroptosis independent of one another is unclear. We find that NFE2L1 can promote ferroptosis resistance, independent of NFE2L2, by maintaining the expression of glutathione peroxidase 4 (GPX4), a key protein that prevents lethal lipid peroxidation. NFE2L2 can also promote ferroptosis resistance but does so through a distinct mechanism that appears independent of GPX4 protein expression. These results suggest that NFE2L1 and NFE2L2 independently regulate ferroptosis.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1073/pnas.2118646119">DOI 10.1073/pnas.2118646119</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/35271393">PubMedID 35271393</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Increased lysosomal biomass is responsible for the resistance of triple-negative breast cancers to CDK4/6 inhibition</span>
  <i>SCIENCE ADVANCES</i>
</span>
<span class="authors">Fassl, A., Brain, C., Abu-Remaileh, M., Stukan, I., Butter, D., Stepien, P., Feit, A. S., Bergholz, J., Michowski, W., Otto, T., Sheng, Q., Loo, A., Michael, W., Tiedt, R., DeAngelis, C., Schiff, R., Jiang, B., Jovanovic, B., Nowak, K., Ericsson, M., Cameron, M., Gray, N., Dillon, D., Zhao, J. J., Sabatini, D. M., Jeselsohn, R., Brown, M., Polyak, K., Sicinski, P.</span>

<span class="details">
   <span class="year">2020</span>; <span class="volume">6 (25)</span>
</span></cite>
    <div class="detail">
      <p class="doi">
    <span>View details for <a href="https://doi.org/10.1126/sciadv.abb2210">DOI 10.1126/sciadv.abb2210</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000542299800037">Web of Science ID 000542299800037</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>The microbiota programs DNA methylation to control intestinal homeostasis and inflammation.</span>
  <i>Nature microbiology</i>
</span>
<span class="authors">Ansari, I., Raddatz, G., Gutekunst, J., Ridnik, M., Cohen, D., Abu-Remaileh, M., Tuganbaev, T., Shapiro, H., Pikarsky, E., Elinav, E., Lyko, F., Bergman, Y.</span>

<span class="details">
   <span class="year">2020</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Although much research has been done on the diversity of the gut microbiome, little is known about how it influences intestinal homeostasis under normal and pathogenic conditions. Epigenetic mechanisms have recently been suggested to operate at the interface between the microbiota and the intestinal epithelium. We performed whole-genome bisulfite sequencing on conventionally raised and germ-free mice, and discovered that exposure to commensal microbiota induced localized DNA methylation changes at regulatory elements, which are TET2/3-dependent. This culminated in the activation of a set of 'early sentinel' response genes to maintain intestinal homeostasis. Furthermore, we demonstrated that exposure to the microbiota in dextran sodium sulfate-induced acute inflammation results in profound DNA methylation and chromatin accessibility changes at regulatory elements, leading to alterations in gene expression programs enriched in colitis- and colon-cancer-associated functions. Finally, by employing genetic interventions, we show that microbiota-induced epigenetic programming is necessary for proper intestinal homeostasis in vivo.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1038/s41564-019-0659-3">DOI 10.1038/s41564-019-0659-3</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/32015497">PubMedID 32015497</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Maintaining Iron Homeostasis Is the Key Role of Lysosomal Acidity for Cell Proliferation.</span>
  <i>Molecular cell</i>
</span>
<span class="authors">Weber, R. A., Yen, F. S., Nicholson, S. P., Alwaseem, H. n., Bayraktar, E. C., Alam, M. n., Timson, R. C., La, K. n., Abu-Remaileh, M. n., Molina, H. n., Birsoy, K. n.</span>

<span class="details">
   <span class="year">2020</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">The lysosome is an acidic multi-functional organelle with roles in macromolecular digestion, nutrient sensing, and signaling. However, why cells require acidic lysosomes to proliferate and which nutrients become limiting under lysosomal dysfunction are unclear. To address this, we performed CRISPR-Cas9-based genetic screens and identified cholesterol biosynthesis and iron uptake as essential metabolic pathways when lysosomal pH is altered. While cholesterol synthesis is only necessary, iron is both necessary and sufficient for cell proliferation under lysosomal dysfunction. Remarkably, iron supplementation restores cell proliferation under both pharmacologic and genetic-mediated lysosomal dysfunction. The rescue was independent of metabolic or signaling changes classically associated with increased lysosomal pH, uncoupling lysosomal function from cell proliferation. Finally, our experiments revealed that lysosomal dysfunction dramatically alters mitochondrial metabolism and hypoxia inducible factor (HIF) signaling due to iron depletion. Altogether, these findings identify iron homeostasis as the key function of lysosomal acidity for cell proliferation.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1016/j.molcel.2020.01.003">DOI 10.1016/j.molcel.2020.01.003</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/31983508">PubMedID 31983508</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Increased lysosomal biomass is responsible for the resistance of triple-negative breast cancers to CDK4/6 inhibition.</span>
  <i>Science advances</i>
</span>
<span class="authors">Fassl, A. n., Brain, C. n., Abu-Remaileh, M. n., Stukan, I. n., Butter, D. n., Stepien, P. n., Feit, A. S., Bergholz, J. n., Michowski, W. n., Otto, T. n., Sheng, Q. n., Loo, A. n., Michael, W. n., Tiedt, R. n., DeAngelis, C. n., Schiff, R. n., Jiang, B. n., Jovanovic, B. n., Nowak, K. n., Ericsson, M. n., Cameron, M. n., Gray, N. n., Dillon, D. n., Zhao, J. J., Sabatini, D. M., Jeselsohn, R. n., Brown, M. n., Polyak, K. n., Sicinski, P. n.</span>

<span class="details">
   <span class="year">2020</span>; <span class="volume">6 (25)</span><span class="pages">: eabb2210</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Inhibitors of cyclin-dependent kinases CDK4 and CDK6 have been approved for treatment of hormone receptor-positive breast cancers. In contrast, triple-negative breast cancers (TNBCs) are resistant to CDK4/6 inhibition. Here, we demonstrate that a subset of TNBC critically requires CDK4/6 for proliferation, and yet, these TNBC are resistant to CDK4/6 inhibition due to sequestration of CDK4/6 inhibitors into tumor cell lysosomes. This sequestration is caused by enhanced lysosomal biogenesis and increased lysosomal numbers in TNBC cells. We developed new CDK4/6 inhibitor compounds that evade the lysosomal sequestration and are efficacious against resistant TNBC. We also show that coadministration of lysosomotropic or lysosome-destabilizing compounds (an antibiotic azithromycin, an antidepressant siramesine, an antimalaria compound chloroquine) renders resistant tumor cells sensitive to currently used CDK4/6 inhibitors. Lastly, coinhibition of CDK2 arrested proliferation of CDK4/6 inhibitor-resistant cells. These observations may extend the use of CDK4/6 inhibitors to TNBCs that are refractory to current anti-CDK4/6 therapies.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1126/sciadv.abb2210">DOI 10.1126/sciadv.abb2210</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/32704543">PubMedID 32704543</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7360435">PubMedCentralID PMC7360435</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Structural basis for the docking of mTORC1 on the lysosomal surface.</span>
  <i>Science (New York, N.Y.)</i>
</span>
<span class="authors">Rogala, K. B., Gu, X., Kedir, J. F., Abu-Remaileh, M., Bianchi, L. F., Bottino, A. M., Dueholm, R., Niehaus, A., Overwijn, D., Fils, A. P., Zhou, S. X., Leary, D., Laqtom, N. N., Brignole, E. J., Sabatini, D. M.</span>

<span class="details">
   <span class="year">2019</span>; <span class="volume">366 (6464)</span><span class="pages">: 468-475</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">The mTORC1 (mechanistic target of rapamycin complex 1) protein kinase regulates growth in response to nutrients and growth factors. Nutrients promote its translocation to the lysosomal surface, where its Raptor subunit interacts with the Rag guanosine triphosphatase (GTPase)-Ragulator complex. Nutrients switch the heterodimeric Rag GTPases among four different nucleotide-binding states, only one of which (RagA/B&#8226;GTP-RagC/D&#8226;GDP) permits mTORC1 association. We used cryo-electron microscopy to determine the structure of the supercomplex of Raptor with Rag-Ragulator at a resolution of 3.2 angstroms. Our findings indicate that the Raptor &#945;-solenoid directly detects the nucleotide state of RagA while the Raptor "claw" threads between the GTPase domains to detect that of RagC. Mutations that disrupted Rag-Raptor binding inhibited mTORC1 lysosomal localization and signaling. By comparison with a structure of mTORC1 bound to its activator Rheb, we developed a model of active mTORC1 docked on the lysosome.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1126/science.aay0166">DOI 10.1126/science.aay0166</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/31601708">PubMedID 31601708</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>WWOX somatic ablation in skeletal muscles alters glucose metabolism.</span>
  <i>Molecular metabolism</i>
</span>
<span class="authors">Abu-Remaileh, M. n., Abu-Remaileh, M. n., Akkawi, R. n., Knani, I. n., Udi, S. n., Pacold, M. E., Tam, J. n., Aqeilan, R. I.</span>

<span class="details">
   <span class="year">2019</span>; <span class="volume">22</span><span class="pages">: 132&#8211;40</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">WWOX, a well-established tumor suppressor, is frequently lost in cancer and plays important roles in DNA damage response and cellular metabolism.We re-analyzed several genome-wide association studies (GWAS) using the Type 2 Diabetes Knowledge Portal website to uncover WWOX's association with metabolic syndrome (MetS). Using several engineered mouse models, we studied the effect of somatic WWOX loss on glucose homeostasis.Several WWOX variants were found to be strongly associated with MetS disorders. In mouse models, somatic ablation of Wwox in skeletal muscle (Wwox&#916;SKM) results in weight gain, glucose intolerance, and insulin resistance. Furthermore, Wwox&#916;SKM mice display reduced amounts of slow-twitch fibers, decreased mitochondrial quantity and activity, and lower glucose oxidation levels. Mechanistically, we found that WWOX physically interacts with the cellular energy sensor AMP-activated protein kinase (AMPK) and that its loss is associated with impaired activation of AMPK, and with significant accumulation of the hypoxia inducible factor 1 alpha (HIF1&#945;) in SKM.Our studies uncover an unforeseen role of the tumor suppressor WWOX in whole-body glucose homeostasis and highlight the intimate relationship between cancer progression and metabolic disorders, particularly obesity and type-2 diabetes.Genetics, Metabolic Syndrome, Diabetes.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1016/j.molmet.2019.01.010">DOI 10.1016/j.molmet.2019.01.010</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/30755385">PubMedID 30755385</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6437662">PubMedCentralID PMC6437662</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>MITO-Tag Mice enable rapid isolation and multimodal profiling of mitochondria from specific cell types in vivo.</span>
  <i>Proceedings of the National Academy of Sciences of the United States of America</i>
</span>
<span class="authors">Bayraktar, E. C., Baudrier, L. n., &#214;zerdem, C. n., Lewis, C. A., Chan, S. H., Kunchok, T. n., Abu-Remaileh, M. n., Cangelosi, A. L., Sabatini, D. M., Birsoy, K. n., Chen, W. W.</span>

<span class="details">
   <span class="year">2019</span>; <span class="volume">116 (1)</span><span class="pages">: 303&#8211;12</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Mitochondria are metabolic organelles that are essential for mammalian life, but the dynamics of mitochondrial metabolism within mammalian tissues in vivo remains incompletely understood. While whole-tissue metabolite profiling has been useful for studying metabolism in vivo, such an approach lacks resolution at the cellular and subcellular level. In vivo methods for interrogating organellar metabolites in specific cell types within mammalian tissues have been limited. To address this, we built on prior work in which we exploited a mitochondrially localized 3XHA epitope tag (MITO-Tag) for the fast isolation of mitochondria from cultured cells to generate MITO-Tag Mice. Affording spatiotemporal control over MITO-Tag expression, these transgenic animals enable the rapid, cell-type-specific immunoisolation of mitochondria from tissues, which we verified using a combination of proteomic and metabolomic approaches. Using MITO-Tag Mice and targeted and untargeted metabolite profiling, we identified changes during fasted and refed conditions in a diverse array of mitochondrial metabolites in hepatocytes and found metabolites that behaved differently at the mitochondrial versus whole-tissue level. MITO-Tag Mice should have utility for studying mitochondrial physiology, and our strategy should be generally applicable for studying other mammalian organelles in specific cell types in vivo.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1073/pnas.1816656115">DOI 10.1073/pnas.1816656115</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/30541894">PubMedID 30541894</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6320505">PubMedCentralID PMC6320505</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>High-fat diet enhances stemness and tumorigenicity of intestinal progenitors (vol 531, pg 53, 2016)</span>
  <i>NATURE</i>
</span>
<span class="authors">Beyaz, S., Mana, M. D., Roper, J., Kedrin, D., Saadatpour, A., Hong, S., Bauer-Rowe, K. E., Xifaras, M. E., Akkad, A., Arias, E., Pinello, L., Katz, Y., Shinagare, S., Abu-Remaileh, M., Mihaylova, M. M., Lamming, D. W., Dogum, R., Guo, G., Bell, G. W., Selig, M., Nielsen, G., Gupta, N., Ferrone, C. R., Deshpande, V., Yuan, G., Orkin, S. H., Sabatini, D. M., Yilmaz, O. H.</span>

<span class="details">
   <span class="year">2018</span>; <span class="volume">560 (7717)</span><span class="pages">: E26</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">In Fig. 4e of this Article, the labels for 'Control' and 'HFD' were reversed ('Control' should have been labelled blue rather than purple, and 'HFD' should have been labelled purple rather than blue). Similarly, in Fig. 4f of this Article, the labels for 'V' and 'GW' were reversed ('V' should have been labelled blue rather than purple, and 'GW' should have been labelled purple instead of blue). The original figure has been corrected online.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1038/s41586-018-0187-y">DOI 10.1038/s41586-018-0187-y</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000441115200013">Web of Science ID 000441115200013</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/29849139">PubMedID 29849139</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Histidine catabolism is a major determinant of methotrexate sensitivity.</span>
  <i>Nature</i>
</span>
<span class="authors">Kanarek, N. n., Keys, H. R., Cantor, J. R., Lewis, C. A., Chan, S. H., Kunchok, T. n., Abu-Remaileh, M. n., Freinkman, E. n., Schweitzer, L. D., Sabatini, D. M.</span>

<span class="details">
   <span class="year">2018</span>; <span class="volume">559 (7715)</span><span class="pages">: 632&#8211;36</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">The chemotherapeutic drug methotrexate inhibits the enzyme dihydrofolate reductase1, which generates tetrahydrofolate, an essential cofactor in nucleotide synthesis2. Depletion of tetrahydrofolate causes cell death by suppressing DNA and RNA production3. Although methotrexate is widely used as an anticancer agent and is the subject of over a thousand ongoing clinical trials4, its high toxicity often leads to the premature termination of its use, which reduces its potential efficacy5. To identify genes that modulate the response of cancer cells to methotrexate, we performed a CRISPR-Cas9-based screen6,7. This screen yielded FTCD, which encodes an enzyme-formimidoyltransferase cyclodeaminase-that is required for the catabolism of the amino acid histidine8, a process that has not&#160;previously been linked to methotrexate sensitivity. In cultured cancer cells, depletion of several genes in the histidine degradation&#160;pathway markedly decreased sensitivity to methotrexate. Mechanistically, histidine catabolism drains the cellular pool of tetrahydrofolate, which is particularly detrimental to methotrexate-treated cells. Moreover, expression of the rate-limiting enzyme in histidine catabolism is associated with methotrexate sensitivity in cancer cell lines and with survival rate in patients. In vivo dietary supplementation of histidine increased flux through the histidine degradation pathway and enhanced the sensitivity of leukaemia xenografts to methotrexate. The histidine degradation pathway markedly influences the sensitivity of cancer cells to methotrexate and may be exploited to improve methotrexate efficacy through a simple dietary intervention.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1038/s41586-018-0316-7">DOI 10.1038/s41586-018-0316-7</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/29995852">PubMedID 29995852</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6082631">PubMedCentralID PMC6082631</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>NUFIP1 is a ribosome receptor for starvation-induced ribophagy.</span>
  <i>Science (New York, N.Y.)</i>
</span>
<span class="authors">Wyant, G. A., Abu-Remaileh, M. n., Frenkel, E. M., Laqtom, N. N., Dharamdasani, V. n., Lewis, C. A., Chan, S. H., Heinze, I. n., Ori, A. n., Sabatini, D. M.</span>

<span class="details">
   <span class="year">2018</span>; <span class="volume">360 (6390)</span><span class="pages">: 751&#8211;58</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">The lysosome degrades and recycles macromolecules, signals to the master growth regulator mTORC1 [mechanistic target of rapamycin (mTOR) complex 1], and is associated with human disease. We performed quantitative proteomic analyses of rapidly isolated lysosomes and found that nutrient levels and mTOR dynamically modulate the lysosomal proteome. Upon mTORC1 inhibition, NUFIP1 (nuclear fragile X mental retardation-interacting protein 1) redistributes from the nucleus to autophagosomes and lysosomes. Upon these conditions, NUFIP1 interacts with ribosomes and delivers them to autophagosomes by directly binding to microtubule-associated proteins 1A/1B light chain 3B (LC3B). The starvation-induced degradation of ribosomes via autophagy (ribophagy) depends on the capacity of NUFIP1 to bind LC3B and promotes cell survival. We propose that NUFIP1 is a receptor for the selective autophagy of ribosomes.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1126/science.aar2663">DOI 10.1126/science.aar2663</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/29700228">PubMedID 29700228</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6020066">PubMedCentralID PMC6020066</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Identification of a transporter complex responsible for the cytosolic entry of nitrogen-containing bisphosphonates.</span>
  <i>eLife</i>
</span>
<span class="authors">Yu, Z. n., Surface, L. E., Park, C. Y., Horlbeck, M. A., Wyant, G. A., Abu-Remaileh, M. n., Peterson, T. R., Sabatini, D. M., Weissman, J. S., O'Shea, E. K.</span>

<span class="details">
   <span class="year">2018</span>; <span class="volume">7</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Nitrogen-containing-bisphosphonates (N-BPs) are a class of drugs widely prescribed to treat osteoporosis and other bone-related diseases. Although previous studies have established that N-BPs function by inhibiting the mevalonate pathway in osteoclasts, the mechanism by which N-BPs enter the cytosol from the extracellular space to reach their molecular target is not understood. Here, we implemented a CRISPRi-mediated genome-wide screen and identified SLC37A3 (solute carrier family 37 member A3) as a gene required for the action of N-BPs in mammalian cells. We observed that SLC37A3 forms a complex with ATRAID (all-trans retinoic acid-induced differentiation factor), a previously identified genetic target of N-BPs. SLC37A3 and ATRAID localize to lysosomes and are required for releasing N-BP molecules that have trafficked to lysosomes through fluid-phase endocytosis into the cytosol. Our results elucidate the route by which N-BPs are delivered to their molecular target, addressing a key aspect of the mechanism of action of N-BPs that may have significant clinical relevance.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.7554/eLife.36620">DOI 10.7554/eLife.36620</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/29745899">PubMedID 29745899</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6021172">PubMedCentralID PMC6021172</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Fasting Activates Fatty Acid Oxidation to Enhance Intestinal Stem Cell Function during Homeostasis and Aging.</span>
  <i>Cell stem cell</i>
</span>
<span class="authors">Mihaylova, M. M., Cheng, C. W., Cao, A. Q., Tripathi, S. n., Mana, M. D., Bauer-Rowe, K. E., Abu-Remaileh, M. n., Clavain, L. n., Erdemir, A. n., Lewis, C. A., Freinkman, E. n., Dickey, A. S., La Spada, A. R., Huang, Y. n., Bell, G. W., Deshpande, V. n., Carmeliet, P. n., Katajisto, P. n., Sabatini, D. M., Yilmaz, &#214;. H.</span>

<span class="details">
   <span class="year">2018</span>; <span class="volume">22 (5)</span><span class="pages">: 769&#8211;78.e4</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Diet has a profound effect on tissue regeneration in diverse organisms, and low caloric states such as intermittent fasting have beneficial effects on organismal health and age-associated loss of tissue function. The role of adult stem and progenitor cells in responding to short-term fasting and whether such responses improve regeneration are not well studied. Here we show that a 24&#160;hr fast augments intestinal stem cell (ISC) function in young and aged mice by inducing a fatty acid oxidation (FAO) program and that pharmacological activation of this program mimics many effects of fasting. Acute genetic disruption of Cpt1a, the rate-limiting enzyme in FAO, abrogates ISC-enhancing effects of fasting, but long-term Cpt1a deletion decreases ISC numbers and function,&#160;implicating a role for FAO in ISC maintenance. These&#160;findings highlight a role for FAO in mediating pro-regenerative effects of fasting in intestinal biology, and they may represent a viable strategy for enhancing intestinal regeneration.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1016/j.stem.2018.04.001">DOI 10.1016/j.stem.2018.04.001</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/29727683">PubMedID 29727683</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5940005">PubMedCentralID PMC5940005</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>mTORC1 Activator SLC38A9 Is Required to Efflux Essential Amino Acids from Lysosomes and Use Protein as a Nutrient.</span>
  <i>Cell</i>
</span>
<span class="authors">Wyant, G. A., Abu-Remaileh, M. n., Wolfson, R. L., Chen, W. W., Freinkman, E. n., Danai, L. V., Vander Heiden, M. G., Sabatini, D. M.</span>

<span class="details">
   <span class="year">2017</span>; <span class="volume">171 (3)</span><span class="pages">: 642&#8211;54.e12</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">The mTORC1 kinase is a master growth regulator that senses many environmental cues, including amino acids. Activation of mTORC1 by arginine requires SLC38A9, a poorly understood lysosomal membrane protein with homology to amino acid transporters. Here, we validate that SLC38A9 is an arginine sensor for the mTORC1 pathway, and we uncover an unexpectedly central role for SLC38A9 in amino acid homeostasis. SLC38A9 mediates the transport, in an arginine-regulated fashion, of many essential amino acids out of lysosomes, including leucine, which mTORC1 senses through the cytosolic Sestrin proteins. SLC38A9 is necessary for leucine generated via lysosomal proteolysis to exit lysosomes and activate mTORC1. Pancreatic cancer cells, which use macropinocytosed protein as a nutrient source, require SLC38A9 to form tumors. Thus, through SLC38A9, arginine serves as a lysosomal messenger that couples mTORC1 activation to the release from lysosomes of the essential amino acids needed to drive cell growth.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1016/j.cell.2017.09.046">DOI 10.1016/j.cell.2017.09.046</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/29053970">PubMedID 29053970</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5704964">PubMedCentralID PMC5704964</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Physiologic Medium Rewires Cellular Metabolism and Reveals Uric Acid as an Endogenous Inhibitor of UMP Synthase.</span>
  <i>Cell</i>
</span>
<span class="authors">Cantor, J. R., Abu-Remaileh, M. n., Kanarek, N. n., Freinkman, E. n., Gao, X. n., Louissaint, A. n., Lewis, C. A., Sabatini, D. M.</span>

<span class="details">
   <span class="year">2017</span>; <span class="volume">169 (2)</span><span class="pages">: 258&#8211;72.e17</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">A complex interplay of environmental factors impacts the metabolism of human cells, but neither traditional culture media nor mouse plasma mimic the metabolite composition of human plasma. Here, we developed a culture medium with polar metabolite concentrations comparable to those of human plasma (human plasma-like medium [HPLM]). Culture in HPLM, relative to that in traditional media, had widespread effects on cellular metabolism, including on the metabolome, redox state, and glucose utilization. Among the most prominent was an inhibition of de novo pyrimidine synthesis-an effect traced to uric acid, which is 10-fold higher in the blood of humans than of mice and other non-primates. We find that uric acid directly inhibits uridine monophosphate synthase (UMPS) and consequently reduces the sensitivity of cancer cells to the chemotherapeutic agent 5-fluorouracil. Thus, media that better recapitulates the composition of human plasma reveals unforeseen metabolic wiring and regulation, suggesting that HPLM should be of broad utility.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1016/j.cell.2017.03.023">DOI 10.1016/j.cell.2017.03.023</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/28388410">PubMedID 28388410</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5421364">PubMedCentralID PMC5421364</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>KICSTOR recruits GATOR1 to the lysosome and is necessary for nutrients to regulate mTORC1.</span>
  <i>Nature</i>
</span>
<span class="authors">Wolfson, R. L., Chantranupong, L. n., Wyant, G. A., Gu, X. n., Orozco, J. M., Shen, K. n., Condon, K. J., Petri, S. n., Kedir, J. n., Scaria, S. M., Abu-Remaileh, M. n., Frankel, W. N., Sabatini, D. M.</span>

<span class="details">
   <span class="year">2017</span>; <span class="volume">543 (7645)</span><span class="pages">: 438&#8211;42</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">The mechanistic target of rapamycin complex 1 (mTORC1) is a central regulator of cell growth that responds to diverse environmental signals and is deregulated in many human diseases, including cancer and epilepsy. Amino acids are a key input to this system, and act through the Rag GTPases to promote the translocation of mTORC1 to the lysosomal surface, its site of activation. Multiple protein complexes regulate the Rag GTPases in response to amino acids, including GATOR1, a GTPase activating protein for RAGA, and GATOR2, a positive regulator of unknown molecular function. Here we identify a protein complex (KICSTOR) that is composed of four proteins, KPTN, ITFG2, C12orf66 and SZT2, and that is required for amino acid or glucose deprivation to inhibit mTORC1 in cultured human cells. In mice that lack SZT2, mTORC1 signalling is increased in several tissues, including in neurons in the brain. KICSTOR localizes to lysosomes; binds and recruits GATOR1, but not GATOR2, to the lysosomal surface; and is necessary for the interaction of GATOR1 with its substrates, the Rag GTPases, and with GATOR2. Notably, several KICSTOR components are mutated in neurological diseases associated with mutations that lead to hyperactive mTORC1 signalling. Thus, KICSTOR is a lysosome-associated negative regulator of mTORC1 signalling, which, like GATOR1, is mutated in human disease.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1038/nature21423">DOI 10.1038/nature21423</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/28199306">PubMedID 28199306</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5360989">PubMedCentralID PMC5360989</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Embryonic Stem Cell (ES)-Specific Enhancers Specify the Expression Potential of ES Genes in Cancer.</span>
  <i>PLoS genetics</i>
</span>
<span class="authors">Aran, D., Abu-Remaileh, M., Levy, R., Meron, N., Toperoff, G., Edrei, Y., Bergman, Y., Hellman, A.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">12 (2)</span><span class="pages">: e1005840</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Cancers often display gene expression profiles resembling those of undifferentiated cells. The mechanisms controlling these expression programs have yet to be identified. Exploring transcriptional enhancers throughout hematopoietic cell development and derived cancers, we uncovered a novel class of regulatory epigenetic mutations. These epimutations are particularly enriched in a group of enhancers, designated ES-specific enhancers (ESSEs) of the hematopoietic cell lineage. We found that hematopoietic ESSEs are prone to DNA methylation changes, indicative of their chromatin activity states. Strikingly, ESSE methylation is associated with gene transcriptional activity in cancer. Methylated ESSEs are hypermethylated in cancer relative to normal somatic cells and co-localized with silenced genes, whereas unmethylated ESSEs tend to be hypomethylated in cancer and associated with reactivated genes. Constitutive or hematopoietic stem cell-specific enhancers do not show these trends, suggesting selective reactivation of ESSEs in cancer. Further analyses of a hypomethylated ESSE downstream to the VEGFA gene revealed a novel regulatory circuit affecting VEGFA transcript levels across cancers and patients. We suggest that the discovered enhancer sites provide a framework for reactivation of ES genes in cancer. </p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1371/journal.pgen.1005840">DOI 10.1371/journal.pgen.1005840</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26886256">PubMedID 26886256</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4757527">PubMedCentralID PMC4757527</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>A PHGDH inhibitor reveals coordination of serine synthesis and one-carbon unit fate.</span>
  <i>Nature chemical biology</i>
</span>
<span class="authors">Pacold, M. E., Brimacombe, K. R., Chan, S. H., Rohde, J. M., Lewis, C. A., Swier, L. J., Possemato, R. n., Chen, W. W., Sullivan, L. B., Fiske, B. P., Cho, S. n., Freinkman, E. n., Birsoy, K. n., Abu-Remaileh, M. n., Shaul, Y. D., Liu, C. M., Zhou, M. n., Koh, M. J., Chung, H. n., Davidson, S. M., Luengo, A. n., Wang, A. Q., Xu, X. n., Yasgar, A. n., Liu, L. n., Rai, G. n., Westover, K. D., Vander Heiden, M. G., Shen, M. n., Gray, N. S., Boxer, M. B., Sabatini, D. M.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">12 (6)</span><span class="pages">: 452&#8211;58</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Serine is both a proteinogenic amino acid and the source of one-carbon units essential for de novo purine and deoxythymidine synthesis. In the canonical pathway of glucose-derived serine synthesis, Homo sapiens phosphoglycerate dehydrogenase (PHGDH) catalyzes the first, rate-limiting step. Genetic loss of PHGDH is toxic toward PHGDH-overexpressing breast cancer cell lines even in the presence of exogenous serine. Here, we used a quantitative high-throughput screen to identify small-molecule PHGDH inhibitors. These compounds reduce the production of glucose-derived serine in cells and suppress the growth of PHGDH-dependent cancer cells in culture and in orthotopic xenograft tumors. Surprisingly, PHGDH inhibition reduced the incorporation into nucleotides of one-carbon units from glucose-derived and exogenous serine. We conclude that glycolytic serine synthesis coordinates the use of one-carbon units from endogenous and exogenous serine in nucleotide synthesis, and we suggest that one-carbon unit wasting thus may contribute to the efficacy of PHGDH inhibitors in vitro and in vivo.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1038/nchembio.2070">DOI 10.1038/nchembio.2070</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/27110680">PubMedID 27110680</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4871733">PubMedCentralID PMC4871733</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>High-fat diet enhances stemness and tumorigenicity of intestinal progenitors.</span>
  <i>Nature</i>
</span>
<span class="authors">Beyaz, S. n., Mana, M. D., Roper, J. n., Kedrin, D. n., Saadatpour, A. n., Hong, S. J., Bauer-Rowe, K. E., Xifaras, M. E., Akkad, A. n., Arias, E. n., Pinello, L. n., Katz, Y. n., Shinagare, S. n., Abu-Remaileh, M. n., Mihaylova, M. M., Lamming, D. W., Dogum, R. n., Guo, G. n., Bell, G. W., Selig, M. n., Nielsen, G. P., Gupta, N. n., Ferrone, C. R., Deshpande, V. n., Yuan, G. C., Orkin, S. H., Sabatini, D. M., Yilmaz, &#214;. H.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">531 (7592)</span><span class="pages">: 53&#8211;58</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Little is known about how pro-obesity diets regulate tissue stem and progenitor cell function. Here we show that high-fat diet (HFD)-induced obesity augments the numbers and function of Lgr5(+) intestinal stem cells of the mammalian intestine. Mechanistically, a HFD induces a robust peroxisome proliferator-activated receptor delta (PPAR-&#948;) signature in intestinal stem cells and progenitor cells (non-intestinal stem cells), and pharmacological activation of PPAR-&#948; recapitulates the effects of a HFD on these cells. Like a HFD, ex vivo treatment of intestinal organoid cultures with fatty acid constituents of the HFD enhances the self-renewal potential of these organoid bodies in a PPAR-&#948;-dependent manner. Notably, HFD- and agonist-activated PPAR-&#948; signalling endow organoid-initiating capacity to progenitors, and enforced PPAR-&#948; signalling permits these progenitors to form in vivo tumours after loss of the tumour suppressor Apc. These findings highlight how diet-modulated PPAR-&#948; activation alters not only the function of intestinal stem and progenitor cells, but also their capacity to initiate tumours. </p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1038/nature17173">DOI 10.1038/nature17173</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26935695">PubMedID 26935695</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4846772">PubMedCentralID PMC4846772</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Chronic inflammation induces a novel epigenetic program that is conserved in intestinal adenomas and in colorectal cancer.</span>
  <i>Cancer research</i>
</span>
<span class="authors">Abu-Remaileh, M., Bender, S., Raddatz, G., Ansari, I., Cohen, D., Gutekunst, J., Musch, T., Linhart, H., Breiling, A., Pikarsky, E., Bergman, Y., Lyko, F.</span>

<span class="details">
   <span class="year">2015</span>; <span class="volume">75 (10)</span><span class="pages">: 2120-30</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Chronic inflammation represents a major risk factor for tumor formation, but the underlying mechanisms have remained largely unknown. Epigenetic mechanisms can record the effects of environmental challenges on the genome level and could therefore play an important role in the pathogenesis of inflammation-associated tumors. Using single-base methylation maps and transcriptome analyses of a colitis-induced mouse colon cancer model, we identified a novel epigenetic program that is characterized by hypermethylation of DNA methylation valleys that are characterized by low CpG density and active chromatin marks. This program is conserved and functional in mouse intestinal adenomas and results in silencing of active intestinal genes that are involved in gastrointestinal homeostasis and injury response. Further analyses reveal that the program represents a prominent feature of human colorectal cancer and can be used to correctly classify colorectal cancer samples with high accuracy. Together, our results show that inflammatory signals establish a novel epigenetic program that silences a specific set of genes that contribute to inflammation-induced cellular transformation.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1158/0008-5472.CAN-14-3295">DOI 10.1158/0008-5472.CAN-14-3295</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/25808873">PubMedID 25808873</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>An Essential Role of the Mitochondrial Electron Transport Chain in Cell Proliferation Is to Enable Aspartate Synthesis.</span>
  <i>Cell</i>
</span>
<span class="authors">Birsoy, K. n., Wang, T. n., Chen, W. W., Freinkman, E. n., Abu-Remaileh, M. n., Sabatini, D. M.</span>

<span class="details">
   <span class="year">2015</span>; <span class="volume">162 (3)</span><span class="pages">: 540&#8211;51</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">The mitochondrial electron transport chain (ETC) enables many metabolic processes, but why its inhibition suppresses cell proliferation is unclear. It is also not well understood why pyruvate supplementation allows cells lacking ETC function to proliferate. We used a CRISPR-based genetic screen to identify genes whose loss sensitizes human cells to phenformin, a complex I inhibitor. The screen yielded GOT1, the cytosolic aspartate aminotransferase, loss of which kills cells upon ETC inhibition. GOT1 normally consumes aspartate to transfer electrons into mitochondria, but, upon ETC inhibition, it reverses to generate aspartate in the cytosol, which partially compensates for the loss of mitochondrial aspartate synthesis. Pyruvate stimulates aspartate synthesis in a GOT1-dependent fashion, which is required for pyruvate to rescue proliferation of cells with ETC dysfunction. Aspartate supplementation or overexpression of an aspartate transporter allows cells without ETC activity to proliferate. Thus, enabling aspartate synthesis is an essential role of the ETC in cell proliferation.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1016/j.cell.2015.07.016">DOI 10.1016/j.cell.2015.07.016</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26232224">PubMedID 26232224</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4522279">PubMedCentralID PMC4522279</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Aberrant DNA methylation in ES cells.</span>
  <i>PloS one</i>
</span>
<span class="authors">Ludwig, G., Nejman, D., Hecht, M., Orlanski, S., Abu-Remaileh, M., Yanuka, O., Sandler, O., Marx, A., Roberts, D., Benvenisty, N., Bergman, Y., Mendelsohn, M., Cedar, H.</span>

<span class="details">
   <span class="year">2014</span>; <span class="volume">9 (5)</span><span class="pages">: e96090</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Both mouse and human embryonic stem cells can be differentiated in vitro to produce a variety of somatic cell types. Using a new developmental tracing approach, we show that these cells are subject to massive aberrant CpG island de novo methylation that is exacerbated by differentiation in vitro. Bioinformatics analysis indicates that there are two distinct forms of abnormal de novo methylation, global as opposed to targeted, and in each case the resulting pattern is determined by molecular rules correlated with local pre-existing histone modification profiles. Since much of the abnormal methylation generated in vitro appears to be stably maintained, this modification may inhibit normal differentiation and could predispose to cancer if cells are used for replacement therapy. Excess CpG island methylation is also observed in normal placenta, suggesting that this process may be governed by an inherent program. </p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1371/journal.pone.0096090">DOI 10.1371/journal.pone.0096090</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/24852222">PubMedID 24852222</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4031077">PubMedCentralID PMC4031077</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>Oct-3/4 regulates stem cell identity and cell fate decisions by modulating Wnt/&#946;-catenin signalling.</span>
  <i>The EMBO journal</i>
</span>
<span class="authors">Abu-Remaileh, M., Gerson, A., Farago, M., Nathan, G., Alkalay, I., Zins Rousso, S., Gur, M., Fainsod, A., Bergman, Y.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">29 (19)</span><span class="pages">: 3236-48</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">Although the transcriptional regulatory events triggered by Oct-3/4 are well documented, understanding the proteomic networks that mediate the diverse functions of this POU domain homeobox protein remains a major challenge. Here, we present genetic and biochemical studies that suggest an unexpected novel strategy for Oct-3/4-dependent regulation of embryogenesis and cell lineage determination. Our data suggest that Oct-3/4 specifically interacts with nuclear &#946;-catenin and facilitates its proteasomal degradation, resulting in the maintenance of an undifferentiated, early embryonic phenotype both in Xenopus embryos and embryonic stem (ES) cells. Our data also show that Oct-3/4-mediated control of &#946;-catenin stability has an important function in regulating ES cell motility. Down-regulation of Oct-3/4 increases &#946;-catenin protein levels, enhancing Wnt signalling and initiating invasive cellular activity characteristic of epithelial-mesenchymal transition. Our data suggest a novel mode of regulation by which a delicate balance between &#946;-catenin, Tcf3 and Oct-3/4 regulates maintenance of stem cell identity. Altering the balance between these proteins can direct cell fate decisions and differentiation.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1038/emboj.2010.200">DOI 10.1038/emboj.2010.200</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/20736927">PubMedID 20736927</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2957205">PubMedCentralID PMC2957205</a></span>
  </p>
      
    </div>
</li>
    <li class="publication  article">
  <cite><span class="title">
  <span>De novo DNA methylation promoted by G9a prevents reprogramming of embryonically silenced genes.</span>
  <i>Nature structural &amp; molecular biology</i>
</span>
<span class="authors">Epsztejn-Litman, S., Feldman, N., Abu-Remaileh, M., Shufaro, Y., Gerson, A., Ueda, J., Deplus, R., Fuks, F., Shinkai, Y., Cedar, H., Bergman, Y.</span>

<span class="details">
   <span class="year">2008</span>; <span class="volume">15 (11)</span><span class="pages">: 1176-1183</span>
</span></cite>
    <div class="detail">
      <h4>Abstract</h4>
  <p class="abstract">The pluripotency-determining gene Oct3/4 (also called Pou5f1) undergoes postimplantation silencing in a process mediated by the histone methyltransferase G9a. Microarray analysis now shows that this enzyme may operate as a master regulator that inactivates numerous early-embryonic genes by bringing about heterochromatinization of methylated histone H3K9 and de novo DNA methylation. Genetic studies in differentiating embryonic stem cells demonstrate that a point mutation in the G9a SET domain prevents heterochromatinization but still allows de novo methylation, whereas biochemical and functional studies indicate that G9a itself is capable of bringing about de novo methylation through its ankyrin domain, by recruiting Dnmt3a and Dnmt3b independently of its histone methyltransferase activity. These modifications seem to be programmed for carrying out two separate biological functions: histone methylation blocks target-gene reactivation in the absence of transcriptional repressors, whereas DNA methylation prevents reprogramming to the undifferentiated state.</p>
  <p class="doi">
    <span>View details for <a href="https://doi.org/10.1038/nsmb.1476">DOI 10.1038/nsmb.1476</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/18953337">PubMedID 18953337</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2581722">PubMedCentralID PMC2581722</a></span>
  </p>
      
    </div>
</li>
</ul>
    </div>
    <div id="journalArticlesContent"></div>
    <div id="booksContent"></div>
    <div id="proceedingsContent"></div>
    <div id="presentationsContent"></div>
    <div id="papersContent"></div>
    <div id="paginationContent"></div>
  </div>
  <div class="span4 side-bar">
    <div id="navigatorContent" class="content-section navigator"></div>
    <div id="similarProfilesContent" class="content-section similar-profiles"></div>
    <div id="tagCloudContent" class="content-section tag-cloud"></div>
  </div>
</div>

      </div>
  </div>
</div>
</section>
        </div>
      </main>
      <footer id="footerContent">
        <div class="container">
          <div class="accessibility">
            <a href="http://www.stanford.edu/site/accessibility">Report Accessibility Issues <i class="icon-caret-right"></i></a>

          </div>
        </div>
        <div class="footer-wrapper">
          <div class="container">
            <div class="row-fluid">
              <div class="span12">
                <div class="logo span2">
                  <a href="http://www.stanford.edu/">Stanford University</a>
                </div>
                <ul class="unstyled span10">
                  <li>
                    <a href="http://www.stanford.edu/">SU Home</a>
                  </li>
                  <li>
                    <a href="http://visit.stanford.edu/plan/maps.html">Maps &amp; Directions</a>
                  </li>
                  <li>
                    <a href="http://www.stanford.edu/search/">Search Stanford</a>
                  </li>
                  <li>
                    <a href="http://www.stanford.edu/site/terms.html">Terms of Use</a>
                  </li>
                  <li>
                    <a href="http://www.stanford.edu/site/copyright.html">Copyright Complaints</a>
                  </li>
                </ul>
                <small class="copyright span10">&#169; Stanford University, Stanford, California 94305</small>
              </div>
            </div>
          </div>
        </div>
      </footer>
    </div>
    <script>
      var require = {
        baseUrl: '/scripts;jsessionid=C0D8DFE61E2E8AE78408BBABD06C3B8E.cap-su-capappprd99',
        urlArgs: 'r=9.11.0'
      };

      (function() {
        if (navigator && navigator.platform === 'Win32') {
          var d = document.getElementsByTagName('html')[0];
          d.className = d.className + ' win';
        }
      }());
    </script>
    <script src="/scripts/vendor/requirejs/require.js;jsessionid=C0D8DFE61E2E8AE78408BBABD06C3B8E.cap-su-capappprd99?r=9.11.0"></script>
    <script>
      
      
      /*<!--*/
      
      define('app-config', {"c":{"accessToken":"","baseUrl":"","capApiUrl":"https://profiles.stanford.edu/proxy/api/cap","googleAnalyticsId":"UA-40553629-2","googleAnalytics4Id":"G-X6DK9R58FD","capIntranetPath":"/intranet","capClassicSignInUrl":"https://cap.stanford.edu/profiles/auth/","browseOrgs":{"administrations":[{"name":"Office of External Relations","url":"/browse/office-of-external-relations"},{"name":"Office of VP for University Human Resources","url":"/browse/office-of-vp-for-university-human-resources"},{"name":"Office of Vice President for Business Affairs and Chief Financial Officer","url":"/browse/office-of-vice-president-for-business-affairs-and-chief-financial-officer"},{"name":"Stanford University Libraries","url":"/browse/stanford-university-libraries"},{"name":"Vice Provost and Dean of Research","url":"/browse/vice-provost-and-dean-of-research"},{"name":"Vice Provost for Student Affairs","url":"/browse/vice-provost-for-student-affairs"},{"name":"Vice Provost for Undergraduate Education","url":"/browse/vice-provost-for-undergraduate-education"}],"institutes":[{"name":"Bio-X","url":"/browse/independent-labs-institutes-and-centers-dean-of-research/bio-x"},{"name":"Freeman Spogli Institute for International Studies","url":"/browse/independent-labs-institutes-and-centers-dean-of-research/freeman-spogli-institute-for-international-studies"},{"name":"Institute for Computational and Mathematical Engineering (ICME)","url":"/browse/school-of-engineering/programs-centers-and-institutes/institute-for-computational-and-mathematical-engineering-icme"},{"name":"Institute for Human-Centered Artificial Intelligence (HAI)","url":"/browse/independent-labs-institutes-and-centers-dean-of-research/institute-for-human-centered-artificial-intelligence-hai"},{"name":"Institute for Stem Cell Biology and Regenerative Medicine","url":"/browse/school-of-medicine/institute-for-stem-cell-biology-and-regenerative-medicine"},{"name":"Precourt Institute for Energy","url":"/browse/stanford-doerr-school-of-sustainability/precourt-institute-for-energy"},{"name":"SLAC National Accelerator Laboratory","url":"/browse/slac-national-accelerator-laboratory"},{"name":"Sarafan ChEM-H","url":"/browse/independent-labs-institutes-and-centers-dean-of-research/sarafan-chem-h"},{"name":"Stanford Institute for Economic Policy Research (SIEPR)","url":"/browse/independent-labs-institutes-and-centers-dean-of-research/stanford-institute-for-economic-policy-research-siepr"},{"name":"Stanford PULSE Institute","url":"/browse/independent-labs-institutes-and-centers-dean-of-research/stanford-pulse-institute"},{"name":"Stanford Woods Institute for the Environment","url":"/browse/stanford-doerr-school-of-sustainability/stanford-woods-institute-for-the-environment"},{"name":"Wu Tsai Human Performance Alliance","url":"/browse/independent-labs-institutes-and-centers-dean-of-research/life-science-institutes-independent-labs-institutes-centers-dor/wu-tsai-human-performance-alliance"},{"name":"Wu Tsai Neurosciences Institute","url":"/browse/independent-labs-institutes-and-centers-dean-of-research/wu-tsai-neurosciences-institute"}],"schools":[{"name":"Graduate School of Business","url":"/browse/graduate-school-of-business"},{"name":"Graduate School of Education","url":"/browse/graduate-school-of-education"},{"name":"School of Engineering","url":"/browse/school-of-engineering"},{"name":"School of Humanities and Sciences","url":"/browse/school-of-humanities-and-sciences"},{"name":"School of Medicine","url":"/browse/school-of-medicine"},{"name":"Stanford Doerr School of Sustainability","url":"/browse/stanford-doerr-school-of-sustainability"}]}},"u":{"intranetPathInUrl":"","switchViewPath":"/intranet/monther-abu-remaileh","uCtxET":0}});
      
      
      // Retrieval of appConfig took 0[ms]
      require(['main']);
      
      /*-->*/
    </script>
  </body>
</html>